source/core/index.ts

```diff
@@ -728,10 +728,7 @@ export default class Request extends Duplex implements RequestEvents<Request> {
 
 			// Trigger search params normalization
 			if (options.url.search) {
-				const triggerSearchParameters = '_GOT_INTERNAL_TRIGGER_NORMALIZATION';
-
-				options.url.searchParams.append(triggerSearchParameters, '');
-				options.url.searchParams.delete(triggerSearchParameters);
+				options.url.search = decodeURIComponent(options.url.search.toString());
```

Comment:
```suggestion
				options.url.search = decodeURIComponent(options.url.search);
```

`.search` is already a string.